src/Database/Expression/AggregateExpression.php

```diff
@@ -24,4 +27,64 @@
  */
 class AggregateExpression extends FunctionExpression
 {
+    /**
+     * @var \Cake\Database\Expression\WindowExpression
+     */
+    protected $window;
+
+    /**
+     * Adds an empty `OVER()` window expression.
+     *
+     * If the window expression for this aggregate is already
+     * initialized, this does nothing.
+     *
+     * @return $this
+     */
+    public function over()
```

Comment:
Once someone has created a window how do they get access to it?

Reply:
`AggregateExpression` will implement `WindowInterface` which passes the calls to `$window` and returns `$this` for chaining.